pkg/dismock/dismock_test.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/diamondburned/arikawa/v3/api"
 	"github.com/diamondburned/arikawa/v3/discord"
+	dismockTesting "github.com/mavolin/dismock/v3/internal/testing"
```

Comment:
Don't use camelCase here, as Effective Go discourages it:
> there should be no need for underscores or mixedCaps
Instead either lowercase it, of, if you find that `dismocktesting` is to hard to read, use a shorter alias like `testingutil` or `testutil`.